### PR TITLE
Rewrite installation instructions

### PR DESCRIPTION
**Pull request by cpaxton**
## Motivation and Context

- installation instructions were really long and hard to follow
- i replaced them with a script which downloads everything you need
- we should just point people to that

## Changelog

- moves content into docs/install_workstation.md and points them to script instead

## Types of changes

- [ ] Docs change / refactoring / dependency upgrade
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

# Checklist:

- [ ] I have made changes to existing documentation where needed.
- [ ] I have added tests that show that the PR is functional.
- [ ] I ran `tests/hw_manual_test.py` on hardware and observed that robot behavior looks normal.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -71,76 +71,36 @@ This should install pytorch; if you run into trouble, you may need to edit the i
...
-```
-rviz -d $HOME_ROBOT_ROOT/src/home_robot_hw/launch/mapping_demo.rviz
+HOME_ROBOT_ROOT=$USER/src/home-robot
+CUDA_HOME=/usr/local/cuda-11.7
 ```
 
-#### 3. Download third-party packages
-```
-git submodule update --init --recursive src/home_robot/home_robot/perception/detection/detic/Detic src/third_party/detectron2 src/third_party/contact_graspnet
+Finally, you can run the [install script](install.sh) to download submodules, model checkpoints, and build Detc for open-vocabulary object detection:
```
Comment (JHurricane96) [inline README.md:84]:
```suggestion
Finally, you can run the [install script](install.sh) to download submodules, model checkpoints, and build Detic for open-vocabulary object detection:
```

**Review by JHurricane96 — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -71,76 +71,36 @@ This should install pytorch; if you run into trouble, you may need to edit the i
...
-
-Download Detic checkpoint as per the instructions [on the Detic github page](https://github.com/facebookresearch/Detic):
-
-```bash
-cd $HOME_ROBOT_ROOT/src/home_robot/home_robot/perception/detection/detic/Detic/
-mkdir models
-wget https://dl.fbaipublicfiles.com/detic/Detic_LCOCOI21k_CLIP_SwinB_896b32_4x_ft4x_max-size.pth -O models/Detic_LCOCOI21k_CLIP_SwinB_896b32_4x_ft4x_max-size.pth --no-check-certificate
+conda activate home-robot
+cd $HOME_ROBOT_ROOT
+./install.sh
```
Comment (hello-fazil) [inline README.md:88]: The install script filename pointed here seems to be wrong.
```suggestion
./install_deps.sh
```

**Comment by hello-fazil** [comment]
In the [install script](https://github.com/facebookresearch/home-robot/blob/95c2f75d7c34fd30bd84eddc5e67523c3b442586/install_deps.sh#L45), the habitat-lab in `src/third_party/habitat-lab` submodule will need to be pulled before attempting to habitat-lab and habitat-baseline python packages.

**Comment by hello-fazil** [comment]
Faced some dependency issues while running eval_episode.py.

- Got the error `AttributeError: module 'PIL.Image' has no attribute 'LINEAR'
`, I tried downgrading the Pillow package version to v7.1, which solved this.
-  Further getting the error `ModuleNotFoundError: No module named 'centernet'`, centernet package which is a dependency wasn't installed.

Other than the above pointed issues, everything works fine

**Review by hello-fazil — APPROVED** [review]

**Comment by hello-fazil** [comment]
> @hello-fazil I don't get the same Pillow error as you with a clean install, but I think everything else is fixed. For the Pillow error - if we see it again lets try to track it down?

Yes sure, that sounds good.

**Comment by FUIGUIMURONG** [comment]
> Faced some dependency issues while running eval_episode.py.
> 
> * Got the error `AttributeError: module 'PIL.Image' has no attribute 'LINEAR' `, I tried downgrading the Pillow package version to v7.1, which solved this.
> * Further getting the error `ModuleNotFoundError: No module named 'centernet'`, centernet package which is a dependency wasn't installed.
> 
> Other than the above pointed issues, everything works fine

I also meet second problem, I am wondering how to fix the dependency problem.

**Comment by Eason-S** [comment]
> Faced some dependency issues while running eval_episode.py.
> 
>     * Got the error `AttributeError: module 'PIL.Image' has no attribute 'LINEAR' `, I tried downgrading the Pillow package version to v7.1, which solved this.
> 
>     * Further getting the error `ModuleNotFoundError: No module named 'centernet'`, centernet package which is a dependency wasn't installed.
> 
> 
> Other than the above pointed issues, everything works fine

@hello-fazil Hi I ran into the same problem `ModuleNotFoundError: No module named 'centernet'` when running `eval_baselines_agent.py`, how did you fix that problem?

**Comment by eyucherin** [comment]
Hi! I am also getting the error `ModuleNotFoundError: No module named 'centernet'`. Would really appreciate some help regarding this issue!

**Comment by hello-fazil** [comment]
Hi @Eason-S @eyucherin I ended up Installing `centernet` from [source](https://github.com/xingyizhou/CenterNet) induvidually.
